csrc/custom_all_reduce.cuh

```diff
@@ -23,17 +24,25 @@
 
 namespace vllm {
 
-constexpr int kMaxBlocks = 64;
-// note: we don't want to use atomics for signals because peer atomics are no
-// supported on PCIe links
+constexpr int kMaxBlocks = 36;
+// Counter may overflow, but it's fine since unsigned int overflow is
+// well-defined behavior.
+using FlagType = uint32_t;
+// Note: we don't want to use atomics for signals because peer atomics are no
```

Comment:
do we still need this comments? which signal need to use atomics?

Reply:
removed.